UI/platform-osx.mm

```diff
@@ -308,34 +308,15 @@ MacPermissionStatus CheckPermissionWithPrompt(MacPermissionType type,
 		break;
 	}
 	case kScreenCapture: {
-#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 110000
-		if (@available(macOS 11.0, *)) {
-			permissionResponse = (CGPreflightScreenCaptureAccess()
+		permissionResponse = (CGPreflightScreenCaptureAccess()
+					      ? kPermissionAuthorized
+					      : kPermissionDenied);
+
```

Comment:
Not directly completely related to this PR, but did we go through with raising the minimum runtime version to 11.0 for the 29.1 release? (I'm not against, just would like to know as it would also mean that we'll need to update support documents)

Reply:
The CMake 3.0 rework is based on that - the minimum system version is 11.0 already.

Reply:
Thanks for clarifying!